index.html

```diff
@@ -0,0 +1,145 @@
+                <a href="#"><img id="menu" src="./images/menu.png" alt="Hamburger"></a>
+            </div>
+            <nav class="mobileNav">
+                <ul id="nav">
+                    <li><a href="#">Home</a></li>
+                    <li><a href="./about.html">About</a></li>
+                    <li><a href="#">Blog</a></li>
+                    <li><a href="#">News</a></li>
+                    <li><a href="#">Sponsor</a></li>
+                  </ul>
+            </nav>
+        </nav>
+    </header>
+
+    <section class="learning">
+
+        <div class="content">
+            <div class="title">
+                <span>Female developer spaces</span>
+                <h1>JOIN WOMEN IN TECH ALL OVER THE WORLD AND BUILD A NETWORK OF DEVELOPERS</h1>
+            </div>
+            <div class="sub-section">
+                <p>We have  built a community of amazing women in tech who come together every once in two weeks to share ideas, look for support and mentor each other.</p>
+            </div>
+            <div class="date">
+                <time datetime="2022-01-11T13:53">2022.01.11(Mon) ~ 7PM(UTC +1)</time>
+                <span>Virtual Meeting</span>
+            </div>
+        </div>
+    </section>
+
+    <section class="program" id="program">
+        <div class="main">
+            <h3>Main Program</h3>
+            <hr class="redLine">
+        </div>
+        <div class="lecture">
+            <div class="forum">
+                <i class="fas fa-laptop-code"></i>
+                <h3>Publication</h3>
+                <p>Listen to the speakers from various countries talk about there journey intotech.</p>
+            </div>
+            <div class="forum">
+                <i class="fab fa-phoenix-framework"></i>
+                <h3>Presenting</h3>
+                <p>Listen to the speakers from various countries talk about there journey intotech.</p>
+            </div>
+            <div class="forum">
+                <i class="fas fa-hamburger"></i>
+                <h3>Discussions</h3>
+                <p>Listen to the speakers from various countries talk about there journey intotech.</p>
+            </div>
+            <div class="forum">
+                <i class="fab fa-stumbleupon-circle"></i>
+                <h3>Mentorship</h3>
+                <p>Listen to the speakers from various countries talk about there journey intotech.</p>
+            </div>
+            <div class="forum">
+                <i class="fas fa-laptop-house"></i>
+                <h3>Document</h3>
+                <p>Listen to the speakers from various countries talk about there journey intotech.</p>
+            </div>
+        </div>
+        <div class="main">
+            <h3 class="one">VIEW ALL ACTIVITIES</h3>
+            <hr class="whiteLine">
+        </div>
+        <div class="bttn">
+            <button type="button" class="btn">Join sheCodes dev conference</button>
+        </div>
+    </section>
```

Comment:
- [ ] There is an overlap between the `learning` section and `Program` section. I would highly recommend fixing it to improve the UI design.
- Your design:
<img width="209" alt="image" src="https://user-images.githubusercontent.com/77190203/152359758-852583ae-d950-4846-8cf6-ded297e29099.png">